app/models/miq_request_task.rb

```diff
@@ -85,6 +85,11 @@ def update_request_status
     update_and_notify_parent(:state => req_state, :status => req_status, :message => display_message(msg))
   end
 
+  def update_request_task(values)
+    update_attributes(:options => options.merge(values['options'] || {}))
```

Comment:
Why accept a hash larger than `options` as an input?

Reply:
@bdunne 
We want a single edit_resource function from the REST API that can edit a specific set of columns in the object.

Reply:
It's a bit awkward that a user could send many columns to update, only `options` will be updated, and the response indicates success.